Observatory-Control-System/BLOCKS/block-jira-project-workflow.rst

```diff
@@ -0,0 +1,245 @@
+Although responsible for execution of the task (arguably the most import step), this person does not have any defined responsibilities in the JIRA Workflow itself. 
+However, they do hold the final say of whether a task can be executed on the summit (as a final evaluation is implicit in their role), and have full discretion to transition tickets back to Preparation if they are found to be lacking instruction. 
+
+Overview of Workflow
+--------------------
+Here we provide a bulleted list illustrating an example workflow. 
+
+* Stakeholder requires an observing task to be completed at the summit, and describes the task in an LVV ticket, SITCOM ticket, or Confluence page. 
+* Stakeholder proposes a new BLOCK Jira ticket, providing links to supporting material and assuming responsibility of Proposer. 
+* New BLOCK Jira ticket is reviewed by Commissioning Science team during one of the regular, weekly planning meetings, and accepted/rejected for scheduling. 
+* New BLOCK Jira ticket is given a priority for scheduling and a Commissioning Scientist is Assigned the ticket, taking on the responsibility of Assignee. 
+* Proposer and Assignee work together to prepare BLOCK ticket for execution.
+* BLOCK ticket is marked as READY by Assignee, and added to the queue of tasks to be executed. 
+* BLOCK ticket is planned for execution at the summit by the night planner. 
+* BLOCK ticket is executed at the summit by Observing Specialist.
+* Data is analyzed by the Proposer and ticket is reported on supporting tickets (e.g. SITCOM). 
+* Proposer verifies observing task is completed and transitions ticket to Done to close issue.
+
+BLOCK Ticket Requirements
+=========================
+
+In addition to the standard JIRA ticket fields, proposers are also required to supply the following information: 
+* Component - either be MainTel or AuxTel
+* Original Time Estimation - selectable in the More Fields section of the ticket. 
+* Links to supporting material either tickets (e.g. SITCOM/LVV) or confluence pages
+
+Time estimation is an important part of the planning process.
+Be sure to include both the execution time and any additional time spent configuring or logging the activity in your time estimate. 
+
+Description
+-----------
+
+The description section of a BLOCK ticket must be carefully filled in to ensure success of the BLOCK. 
+BLOCK descriptions walk a very fine line between having just enough information and not too much. 
+They should not read like a novel, too much information is likely to get lost at the pace with which execute at night. 
+They must contain the minimum set of instructions needed to carry out the complete observing task. 
+
+We recommend that at a minimum the description section must contain:
+* Description, high-level overview of test
+* Pre-conditions/requirements to run test, including system status
+* Environmental Constraints on test (e.g. observing conditions/time)
+* Test Execution Instructions
```

Comment:
Should we move this and the next items to the Test Case instead? we don't want to duplicate information right?

Reply:
But who will create the test case? Some people may not be familiar with the test cases in Zephyr. So, duplication might be welcome, since a ComSci will need to transfer that information to the test case. Unless ComSci gets that information directly from proposer and put it in the test case

Reply:
See my comment below.